server/service/hosts.go

```diff
@@ -154,7 +154,20 @@ func (svc *Service) ListHosts(ctx context.Context, opt fleet.HostListOptions) ([
 		opt.MDMBootstrapPackageFilter = nil
 	}
 
-	return svc.ds.ListHosts(ctx, filter, opt)
+	hosts, err := svc.ds.ListHosts(ctx, filter, opt)
+	if err != nil {
+		return nil, err
+	}
+
+	if opt.PopulateSoftware {
+		for _, host := range hosts {
+			if err = svc.ds.LoadHostSoftware(ctx, host, true); err != nil {
```

Comment:
Re: `true`here: IIRC we support the CVE scores only when the user is using Premium.

Reply:
FIxed!